Octokit/Models/Request/RepositoryRequest.cs

```diff
@@ -43,7 +43,7 @@ public class RepositoryRequest : RequestParameters
         /// <value>
         ///  The visibility.
         /// </value>
-        public RepositoryVisibility? Visibility { get; set; }
+        public RepositoryRequestVisibility? Visibility { get; set; }
```

Comment:
Noting this for the release notes: breaking change `RepositoryVisibility` -> `RepositoryRequestVisibility`